app/src/main/java/com/sci/coffeeandroid/feature/menudetails/data/repository/CoffeeDetailRepository.kt

```diff
@@ -0,0 +1,14 @@
+package com.sci.coffeeandroid.feature.menudetails.data.repository
+
+import com.sci.coffeeandroid.feature.menudetails.data.datasource.CoffeeDetailRemoteDataSource
+import com.sci.coffeeandroid.feature.menudetails.data.datasource.CoffeeDetailRemoteDataSourceImpl
+import com.sci.coffeeandroid.feature.menudetails.domain.model.CoffeeModel
+import kotlinx.coroutines.delay
+
+class CoffeeDetailRepository(private val coffeeDetailRemoteDataSource: CoffeeDetailRemoteDataSource) {
+
+    suspend fun getCoffeeDetail(id: Int): Result<CoffeeModel>{
+        delay(600)
```

Comment:
should be write fake loading in fake remote datasource as it is relate with network delay